services/processor/native/service.go

```diff
@@ -69,6 +89,9 @@ func (s *service) ProcessCall(ctx context.Context, input *services.ProcessCallIn
 		}, err
 	}
 
+	start := time.Now()
```

Comment:
Yes. Your link in the playground provides evidence that we need instantiate `start` first because `defer` will only be evaluated on exit, rendering it useless.

Reply:
Turns out that `defer x(y())` would only defer evaluation of `x()` and `y()` is going to be evaluated immediately.